gp_lts_workflow/example_subpkg/shape.py

```diff
@@ -0,0 +1,22 @@
+import astropy.units as u
+
+
+class Rectangle:
+    """Implement a square.
+
+    Parameters
+    ----------
+    x : `~astropy.units;Quantity`
+        x size
+    y : `~astropy.units;Quantity`
+        y size
+    """
+
+    @u.quantity_input
+    def __init__(self, x: u.cm, y: u.cm):
+        self.x = x
```

Comment:
I think I could actually put in negative numbers here, which will result in a negative area. Should we restrict to positive numbers here?